nautobot/core/jobs/__init__.py

```diff
@@ -199,6 +200,12 @@ class ImportObjects(Job):
     )
     csv_data = TextVar(label="CSV Data", required=False)
     csv_file = FileVar(label="CSV File", required=False)
+    roll_back_if_error = BooleanVar(
+        label="Rollback if error",
+        required=False,
+        default=False,
+        description="If an error occurs while importing should all imports be rollback",
```

Comment:
Maybe something like the following?

```suggestion
        label="All or nothing import",
        required=False,
        default=False,
        description="Enable so that if any error occurs while importing, even successfully imported objects will be discarded. Disable if a partial import is acceptable.",
```